### DISCUSSION TITLE
Use multiple networks

### DISCUSSION THREAD

**Discussion by Nico83500** [containrrr/watchtower#919] [Q&A] [answered]
Hi,
I already asked it to Diun because I use it but I would like to try Watchtower also so I post it here. 
I would like to use 2 differents networks :
- An internal network to communicate between the socket proxy and Watchtower container
- An external network to communicate to the web (to get docker hub access, updates, send notif, etc...)
Is there a way to indicate which network to use to external communications ? Or is it possible to add a feature to do that ?
Thank you !

**Comment by simonaronsson** [comment, marked as answer]
Currently we don't have any explicit support for this. I would assume that if only one of the networks has an upstream internet connection, that one would be used for those requests, but if you've already tried it and couldn't get it to work, we should probably add this to the backlog (not promising anything about **where** however 😅 ). 👍🏼

**Reply by Nico83500** [reply]
Thank you ;) I tried it again, monitor only, but I don't receive any notifications. Is there a way to test notifications ? Thank you.

**Reply by piksel** [reply]
Easiest way is to run with `--run-once --debug` which should at least send a startup message if there are no updates. And if it couldn't send notifications it should have logged why.

**Reply by Nico83500** [reply]
Thank you.
I just run with these arguments and I get this log from docker logs (I replaced my personal info for smtp) : 

> time="2021-04-23T11:41:00Z" level=debug
smtp://login:pass@server:port/?auth=Plain&encryption=Auto&fromaddress=name@domain.tld&fromname=Watchtower&starttls=Yes&subject=Watchtower+updates+on+3c3f3bd265b3&toaddresses=name@domain.tld&usehtml=No
time="2021-04-23T11:41:00Z" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."
time="2021-04-23T11:41:01Z" level=debug msg="Retrieving running containers"
time="2021-04-23T11:41:01Z" level=debug msg="There are no additional watchtower containers"
time="2021-04-23T11:41:01Z" level=debug msg="Watchtower HTTP API skipped."
time="2021-04-23T11:41:01Z" level=info msg="Watchtower v0.0.0-unknown\nUsing notifications: smtp\nOnly checking containers using enable label\nScheduling first run: 2021-04-23 16:00:00 +0000 UTC\nNote that the first check will be performed in 4 hours, 18 minutes, 58 seconds"


Perhaps there are other logs ? Thank you.
